ghost/i18n/locales/uk/portal.json

```diff
@@ -0,0 +1,60 @@
+{
+    "{{discount}}% discount": "Знижка {{discount}}%",
+    "{{trialDays}} days free": "{{trialDays}} днів безкоштовно",
```

Comment:
Depending on how many days there are, the term "днiв" (Ukrainian for "days") has different declensions. In order to solve this another variable has to be introduced: 

`"{{trialDays}} days free": "{{trialDays}} {{daysOrDay}} безплатно"`

This will ensure the correct grammatical form is used for different values of trial days in the Ukrainian locale.

Reply:
Went with "Безплатно {{trialDays}} дні(-в)" if it bothers anybody too much we can always change :)